client/state/themes/selectors/is-full-site-editing-theme.ts

```diff
@@ -11,7 +11,7 @@ export function isFullSiteEditingTheme(
 	if ( ! themeId ) {
 		return false;
 	}
-	const theme = getTheme( state, 'wpcom', themeId );
+	const theme = getTheme( state, 'wpcom', themeId ) || getTheme( state, 'wporg', themeId );
```

Comment:
Are we sure that `getThemeTaxonomySlugs` works for dotorg themes? 🤔 This is an example of a dotorg theme payload: https://api.wordpress.org/themes/info/1.1/?action=theme_information&request%5Bfields%5D%5Bextended_author%5D=true&request%5Bslug%5D=astra

Reply:
I confirmed it worked for several FullSiteEditing dotorg themes, but let me check again!

Reply:
I found a couple of things that can be evidence that the `full-site-editing` tag means Full Site Editing compatible. I thought the current implementation was reliable enough! Maybe I should add comments?

>pNEWy-ezX-p2#comment-54325
we are now aligned with Core in using full-site-editing to tag FSE themes in the showcase

>https://developer.wordpress.org/block-editor/getting-started/full-site-editing/
You can find other themes in the directory using the [full-site-editing feature tag](https://wordpress.org/themes/tags/full-site-editing/).

>https://github.com/search?q=repo%3AWordPress%2Fwordpress-develop%20full-site-editing&type=code